package.json

```diff
@@ -8,9 +8,9 @@
     "url": "https://github.com/mooyoul/proxyfront.git"
   },
   "scripts": {
-    "build": "check-engine && rm -Rf dst && tsc && cd src && find . \\( -name '*.json' -o -name '*.html' \\) -type f -exec cp {} ../dst/{} \\;  && cd ..",
-    "pack": "rm -f dst.zip && cp package.json dst/package.json && (cp package-lock.json dst/ || true) && (cp .npmrc dst/.npmrc || true) && cd dst && npm install --cache=../.npm --production && npm ls && zip -rqy ../dst.zip . && cd ..",
-    "pretest": "check-engine && rm -Rf dst && tsc -p ./tsconfig.test.json && cd src && find . -name '*.json' -type f -exec cp {} ../dst/{} \\; && cd ..",
+    "build": "rm -Rf dst && tsc && cd src && find . \\( -name '*.json' -o -name '*.html' \\) -type f -exec cp {} ../dst/{} \\;  && cd ..",
```

Comment:
Why did you remove check-engine? if you want to target to Node.js 12 runtime of Lambda@Edge, just specify engines.node to `^12.10.0`.